core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -629,7 +629,7 @@ oppia.factory('explorationPropertyService', [
       // Validates the given value and returns a boolean stating whether it
       // is valid or not. THIS CAN BE OVERRIDDEN BY SUBCLASSES. The default
       // behavior is to always return true.
-      _isValid: function(value) {
+      _isValid: function() {
```

Comment:
I think while we are catching these we can change them as we go, that way they don't become a larger problem.